about.html

```diff
@@ -0,0 +1,258 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link
+      href="https://cdn.jsdelivr.net/npm/bootstrap@5.2.1/dist/css/bootstrap.min.css"
+      rel="stylesheet"
+    />
+    <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.2.1/dist/js/bootstrap.bundle.min.js"></script>
+    <link rel="stylesheet" href="styles.css" />
+    <title>Web Development Course/About</title>
+  </head>
+  <body>
+    <div class="home-page-main-container">
+      <div class="side-nav open">
+        <ul class="side-nav-items-container">
+          <li class="side-nav-close-btn-container side-nav-size">
+            <img
+              src="images/close_mobile_menu.png"
+              alt="close button"
+              class="side-nav-close-btn"
+            />
+          </li>
+          <li class="side-nav-about-container side-nav-size">
+            <a href="about.html" class="sid-nav-about">About</a>
```

Comment:
![capstone-aboutpage-white-spacing](https://user-images.githubusercontent.com/93039359/198582350-957496cb-a942-4adc-ab80-d7489cfd2174.png)

- [ ] Kindly check my comments on the image above :+1: and kindly fix those things which i mentioned to make your design look similar to the Behance Design :+1: